visual-testing-app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "visual-testing-app",
```

Comment:
It probably makes sense to include this in the yarn workspace, otherwise it needs to be installed separately

Reply:
oh that's the problem I was having, thanks!